.travis.yml

```diff
@@ -17,11 +17,12 @@ env:
     - TEST_SPECIFIC_MODULES=presto-cassandra
```

Comment:
Please make sure your commit message follows https://chris.beams.io/posts/git-commit/.
In your case https://chris.beams.io/posts/git-commit/#capitalize.